registry.json

```diff
@@ -3600,5 +3600,19 @@
     "telegram": "SmaugPool",
     "operatorName": "",
     "github": "SmaugPool"
+  },
+   {
+    "ticker": "PROOF",
+    "poolId": "pool17s9pcsnhgueer24p4lk53ruglr0r8009yxfrqvwgk5gx6r44cpe",
+    "social": {
+      "website": "https://www.proofstakepool.com",
+      "twitter": "",
```

Comment:
Please add a social media presence of some kind

Reply:
Twitter: ProofStakePool